components/KieslerKreis/index.js

```diff
@@ -1,13 +1,14 @@
 import { Radar } from "react-chartjs-2";
 import Chart from "chart.js/auto";
```

Comment:
You don't need that import anymore.

Reply:
actually even if it seems unused it is needed for react-chartjs-2 to work 😊